### PR TITLE
Basic App Refactor

### PR DESCRIPTION
**Pull request by alexplazas**
The purpose of this PR is to define our application in a basic form, with initial definitions of concepts such as our blueprints and routes, database and database connections, and tests.

The app refactor also translates our initial application to a more standard format that is supported and easily integrates with existing tools.

### CODE REVIEW

File: recipeezy/OLD_recipeexy.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# from flask import Flask, session, redirect, url_for, request, render_template
```
Comment (alexplazas) [inline recipeezy/OLD_recipeexy.py:1]: Commented-out code should be removed or resolved in a coming PR

File: recipeezy/__init__.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        # load the test config if passed in
+        app.config.from_mapping(test_config)
+
+    # ensure the instance folder exists
+    try:
+        os.makedirs(app.instance_path)
+    except OSError:
+        pass
+
+    # a simple page that says hello
```
Comment (alexplazas) [inline recipeezy/__init__.py:45]: Commented code should be removed or should indicate follow-up work

File: recipeezy/home.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    # if 'username' in session:
+    #     username = escape(session['username'])
+    # else:
+    #     username = None
+    print("Test", flush=True)
+    if req.form.get('menu', None):
+        return redirect(url_for('home_bp.home'))
+    elif req.form.get('login', None):
+        return redirect(url_for('login_bp.login'))
+    # elif req.form.get('logout', None):
```
Comment (alexplazas) [inline recipeezy/home.py:23]: Commented code should be removed prior to merging

File: recipeezy/static/recipes/Chicken_Provencal [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+1/2 cup green olives
+1 - lemon, cut into eighths
+1 tablespoon capers
+4 sprigs thyme
+1 cup white wine
+- - Rice or baguette
+RECIPE:
+1. Preheat Oven to 400°F
+2. Season the chicken thighs well on all side with salt and pepper.
+
```
Comment (alexplazas) [inline recipeezy/static/recipes/Chicken_Provencal:29]: Consider changing the plaintext recipes to JSON objects

**Review by CSPB-Luis — APPROVED** [review]
+1

File: recipeezy/static/recipes/Chicken_Provencal (hunk 4 above)
Comment (dash2927) [reply recipeezy/static/recipes/Chicken_Provencal:29]: ya, JSON will be in the next commit

**Review by dash2927 — APPROVED** [review]

**Comment by AdhirU** [comment]
Looks good!

**Pull request merged**